locales/en.json

```diff
@@ -477,6 +477,8 @@
     "unlockable": {
       "phaseIntroduction": "We will have a 60-minute window every phase, each featuring only 10 exclusive items.",
       "readyIn": "Your mint will be ready in {0}, DO NOT CLOSE THIS WINDOW",
+      "mintLive": "Free Unlockable Mint Live",
+      "takeMe": " Take Me There!",
```

Comment:
lol is the space forced here? if so it would be better to use `&nbsp;`

Reply:
It's a mistake : )